pkg/config/config_test.go

```diff
@@ -152,9 +152,12 @@ apiVersion: config.kueue.x-k8s.io/v1beta1
 kind: Configuration
 waitForPodsReady:
   enable: true
+  timeout: 50s
```

Comment:
nit question: any reason for changes  in this file?

Reply:
I modified this test data because there are no test cases to verify if overrided parameters are propagated.

Reply:
Sgtm, I see this is what we do also for other fields

Reply:
That's right. Maybe we forgot to add these fields here when we extend waitForPodsReadt feature :)